modules/manager/controller_setup.go

```diff
@@ -406,7 +407,11 @@ func SetupControllers(mgr manager.Manager, c *Config) (map[string]ControllerDef,
 				Scheme:                   mgr.GetScheme(),
 				ClusterCASecretName:      c.ClusterCASecretName,
 				ClusterCASecretNamespace: c.ClusterCASecretNamespace,
-				DevelopmentMode:          c.DevelopmentMode,
+				ClusterCAKeyConfig: secrets.KeyConfig{
+					Type: c.ClusterCAKeyType,
+					Size: c.ClusterCAKeySize,
+				},
```

Comment:
Please consider creating `keyConfig` once and passing it over to all controllers.

Reply:
👍 ee1085baff2321f28da85b742c012ad6828ffc8d